src/Service/Bulk/BulkUpdateUsersAssignmentsStateService.php

```diff
@@ -129,14 +128,14 @@ private function processOperation(BulkOperation $operation, BulkResult $result):
     /**
      * @throws LogicException
      */
-    private function validateStateTransition(BulkOperation $operation): void
+    private function validateStateAttribute(BulkOperation $operation): void
```

Comment:
can we change it to `validateStatusAttribute` as well?

Reply:
Fixed